components/home/mcp-card.tsx

```
@@ -39,17 +43,22 @@ function injectEnvVarsToLocalConfig(localConfig: any, envVars: any[] = [], apiKe
 }
 
 // Recursively mask all sensitive values, including API key fields
-function maskSensitiveLocalConfig(localConfig: any, envVars: any[] = [], reveal: boolean = false, apiKey?: string) {
+function maskSensitiveLocalConfig(
+    localConfig: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)